terraform/modules/data-share-service/rds.tf

```
@@ -48,23 +47,23 @@ resource "aws_rds_cluster_instance" "db_aurora" {
 
 resource "aws_security_group" "rds_postgres_cluster" {
   name_prefix = "${var.environment}-rds-postgres-cluster-"
-  description = "For RDS cluster, inbound access from ECS only"
+  description = "For RDS cluster, inbound access from ECS, bastion host only"
```

```suggestion
  description = "For RDS cluster, inbound access from ECS or bastion host only"
```
Is that what this means?